content/en/docs/other-guides/kubeflow-behind-proxy.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Deploying Kubeflow behind a proxy server"
+description = "Instructions to install and configure Kubeflow behind a proxy server"
+weight = 10
+
++++
+
+## Install Kubeflow behind a proxy server
+
+This guide demonstrates how to install and configure Kubeflow behind a proxy server.
+
+**Note**: These instructions assume that your Docker setup has a [configured
+proxy service](https://docs.docker.com/network/proxy/) on each Kubernetes node.
+
+1. Create an alias for the proxy:
+
+```
```

Comment:
Here and everywhere else in this doc where you've got a code block, let's add `shell` like so:
```suggestion
```shell
```